career-growth.md

```diff
@@ -0,0 +1,69 @@
+# Conscious Career Growth
+
+It used to be the case that people would stay with one employer for their entire careers. This is no longer the case today. It is normal and expected for most people to move from role to role throughout their lives. That said, you are at Packback right now for a reason! Your job with Packback is an alliance that benefits both parties:
+
+* You apply your unique skills, vision and expertise to move Packback forward as a company.
```

Comment:
Since we use an oxford comma in the second bullet point, we should use one here as well. Could we add a comma after `vision` ?